app/services/storage_service.rb

```diff
@@ -8,6 +8,14 @@ def initialize(paas_config_service, paas_instance_name)
     @client = create_client
   end
 
+  def list_files(folder)
+    file_list = []
+    @client.list_objects_v2(bucket: @configuration.bucket_name, prefix: folder).each do |response|
```

Comment:
Is this equivalent?
 
```
@client.list_objects_v2(bucket: @configuration.bucket_name, prefix: folder).map { |response| response.contents.map(&:key) }
```

Reply:
Should be, I am not a huge fan of `map` to Kat disappear 😂 
If you think this is more readable, I will update and run the tests.

Reply:
:scream: I prefer `map` to each as being more functional/idiomatic but purely personal style preference so happy either way :slightly_smiling_face:

Reply:
We'd need to flatten it tho if that how we do it, right? Cause we'd get an array of arrays?